.github/workflows/test-deploy-main.yml

```diff
@@ -325,3 +325,17 @@ jobs:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       - name: Publish to next tag
         run: yarn publish:next
+
+  build-test: 
+    runs-on: ubuntu-latest
```

Comment:
Should have this depend on `publish`?

Reply:
Great catch, [a344a90](https://github.com/aws-amplify/amplify-ui/pull/1747/commits/a344a90bda93877ccf608870d5b8b44f8791c398)